src/eos-dropbox-app

```diff
@@ -47,6 +48,12 @@ def get_default_dropbox_directory():
     if os.path.isdir(default_dir):
         return default_dir
 
+    # No 'Dropbox' directory found, our last attempt will be to look for a Dropbox
+    # team folder, used in 'business' accounts (e.g. 'Dropbox (Endless Team)')
+    team_dir = glob.glob(default_dir + ' (*)')
+    if team_dir:
+        return team_dir[0]
+
```

Comment:
I wonder if looking for a default dir at all is a good idea. If the config is not in place, won't dropbox ask the user again to log in, which, at that point may bring a completely different directory that one that looks like a "default" one?